lib/data/index.js

```diff
@@ -0,0 +1,108 @@
+var romis = require('./romis');
+var _ = require('lodash');
+var expireTime = require('../../config/redis').redisExpirationSeconds;
+var BPromise = require('bluebird');
+
+var keyPrefix = 'JANE_CASEY_';
+
+function formatKey(key){
+  if(key && key.indexOf(keyPrefix)===0) return key;
+
+  return keyPrefix + (key||'invalid');
+}
+
+function resolve(value){
+  return new BPromise(function(resolve){
+    resolve(value);
+  });
+}
+
+
+function setData(key,data){
+  console.log('setting data');
+  key = formatKey(key);
+
+  return romis.set(key,JSON.stringify(data))
+    .then(function(){
+      console.log('setting data expiration: ' +expireTime +' seconds');
+      if(expireTime > 0) {
```

Comment:
So if you don't want an expiration you set the environment variable to 0 or less.